docs/README.md

```diff
@@ -12,19 +12,23 @@ Set up a Postgres database and user. Record the details.
 
 These instructions assume Linux. 
 
+<<<<<<< HEAD
```

Comment:
Is this from a merge conflict and needs cleaning up?

Reply:
Arg thanks. Fixed now.